source/install/config-tls-mattermost.rst

```diff
@@ -42,3 +42,17 @@ The easiest option is to set up TLS on the Mattermost Server, but if you expect
 
 .. note::
    Password-protected certificates are not supported.
+   
+**Configure TLS on NGINX (proxy)**
+ 
+ 1. NGINX acts as a forward proxy to encrypt the traffic between the client and Mattermost server.
+ 2. (Optional) Upstream encryption between NGINX to Mattermost server is allowed.
+ 3. Follow `NGINX's guide on setting up SSL Termination for TCP Upstream Servers <https://docs.nginx.com/nginx/admin-guide/security-controls/terminating-ssl-tcp/>`__.
+ 
+ Other helpful resources:
+ 
+- `NGINX's SSL blog <https://www.nginx.com/blog/nginx-ssl/>`__
+- `NGINX's SSLguide <https://docs.nginx.com/nginx/admin-guide/security-controls/terminating-ssl-http/>`__
+ 
+.. note::
+  Do not set up TLS on Mattermost before before doing so for NGINX. It breaks the connection as the TLS prevents it from successfully communicating with the Mattermost server.
```

Comment:
This is a really helpful note. I propose moving it to the beginning of the section for awareness.

Reply:
done